imagersite/imagersite/urls.py

```diff
@@ -13,9 +13,27 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.contrib import admin
+from django.shortcuts import render
+from django.views.generic.edit import CreateView
+from django.contrib.auth.forms import UserCreationForm
+
+
+def home_view(request):
+    context = {
+        'page_title': 'Home'
+    }
+    return render(request, 'imagersite/index.html', context)
```

Comment:
We should be separating our concerns. Put views into a `views.py` file and then import them.